### ISSUE TITLE
Feature Request: Group Alerts By Repo

### ISSUE THREAD

**Issue by crackalak** [rajbos/GHAzDo-widget#50]
I'd like to be able to see the alerts across the project grouped by repo. 
We can currently see the totals but this doesn't tell us which repo we need to focus on and we still have to load the normal Advanced Security page for each repo.

For example, you could show stacked bar charts for each alert type that shows the number of alerts per day by repo.

**Comment by philcart** [comment]
Yes, would definitely like to see this across all repositories in the project. 

My current company uses a raft of small internally developed .Net applications. Keeping track of things individually is a nightmare.

**Comment by laurajjiang** [comment]
Hi @philcart and @crackalak - have you looked at the built-in security overview dashboard for this need? You can find the documentation at [Security Overview for GitHub Advanced Security](https://learn.microsoft.com/en-us/azure/devops/repos/security/github-advanced-security-security-overview?view=azure-devops) - this will provide you with a summary of active alerts across all projects/repos in your organization.

**Comment by crackalak** [comment]
@laurajjiang I must admit I didn't know this existed until the other day, would still be nice to visualise the change over time across the repos though

**Comment by laurajjiang** [comment]
Heard - we have metrics on the roadmap but no ETA on when we'll be able to add some of this historical data or other metrics to security overview yet.

**Comment by philcart** [comment]
I'd also like to see the information in one spot and not have to navigate to a potentially inaccessible place to view details. We run a couple of DevOps projects and only a couple of people have organisational level viewing.
